website/source/docs/agent/dns.html.markdown

```diff
@@ -198,7 +198,17 @@ Both A and SRV records are supported. SRV records provide the port that a servic
 registered on, enabling clients to avoid relying on well-known ports. SRV records are
 only served if the client specifically requests them.
 
-### UDP Based DNS Queries
+## Agent Query Lookup
+
+The agent query is designed to return IP information about the local Consul 
+agent. The IP information returned is the DNS answer for the local node name.
+This does in fact send a DNS query from the local instance to the Consul Server
+Quorum. The result is the IP informaiton that any other service would receive
```

Comment:
Typo: Information

Reply:
Done in 65f112ae5f487d6f8574c2e731c5a199d0b60589 over in #3805.